freightliner_cascadia_2012_dot_10002/VehicleConfigParams.yaml

```diff
@@ -102,6 +102,12 @@ camera_drivers:
 # Used for interpreting traffic rules in world model instances
 vehicle_participant_type: "vehicle:truck"
 
+#BSM Message ID for Simulation Testing (Will be converted to uint8_t in bsm_generator)
```

Comment:
This comment is not really accurate. The integer value will be converted to a 4 element array of uint8_t where each byte of the parameter becomes one element of the array. This is not the same as truncating the array to 1 byte represented by uint8_t. 
Update comment for clarity